packages/block-editor/src/components/block-list/block-invalid-warning.js

```diff
@@ -101,8 +101,7 @@ export default function BlockInvalidWarning( { clientId } ) {
 			<Warning
 				actions={ [
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
The invalid warning can be seen when a block's content is invalid after saving the page and then returning to it.

Before | After     |
| :-------- | :------- |
| ![block validation 36](https://github.com/user-attachments/assets/70a38cc5-ae43-406f-8a1c-8fe7ccc4ba24)| ![block invalidation 40px](https://github.com/user-attachments/assets/4e9687e4-9def-4697-8446-fccc2e437d65)|